.github/workflows/build-and-upload-bottles.yml

```diff
@@ -0,0 +1,79 @@
+name: Build and Upload Homebrew Bottles
+
+on:
+  push:
+    branches:
+      - master
+      
+env:
+  KURTOSIS_CLI_FORMULA: kurtosis-tech/tap/kurtosis-cli
+  SEMVER_REGEX: '(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    outputs:
+      upload_url: ${{ steps.create_release.outputs.upload_url }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Attempt to get version from latest commit message
+        uses: actions-ecosystem/action-regex-match@v2
+        id: get-version
+        with:
+          regex: ${{ env.SEMVER_REGEX }}
+          text: ${{ github.event.head_commit.message }}
+
+      # We assume that the commit messages that are pushed to tap repo by goreleaser contain the release version
+      # Thus, if this ever changes or an accidental commit was made we need to fail loudly, in order to not attempt building the bottles
+      - name: Check if version was parsed from commit message and fail loudly if not
+        if: ${{ steps.get-version.outputs.match == '' }}
+        # This action allows using Github API directly, so we can use the core libraries fail function to fail loudly
+        # https://timheuer.com/blog/manually-force-a-failure-in-github-action-step/
+        uses: actions/github-script@v3
+        with: 
+          script: |
+            core.setFailed('Unable to parse semantic version from latest tap commit message.')
```

Comment:
interesting, I was wondering if you could do an "exit 1" or something but this is alright too

https://stackoverflow.com/questions/57903836/how-to-fail-a-job-in-github-actions

Reply:
Yep, `exit 1` works! [`core.setFailed()` here is simply a shortcut for `::error` which shows a message and `exit 1` which exits](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#using-workflow-commands-to-access-toolkit-functions).